external/fv3fit/tests/emulation/test_data_transforms.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+import numpy as np
+import tensorflow as tf
+import xarray as xr
+from fv3fit.emulation.data import transforms as xfm
```

Comment:
nit: Some style guides (e.g. Google) disallow this kind of nonstandard renaming and I don't think `transforms` is an overly long name to use below. Is there some reason we need a three letter abbreviation for this?

```suggestion
from fv3fit.emulation.data import transforms
```

Reply:
Same story as above suggestion.